### ISSUE TITLE
linker errors when building _tf

### ISSUE THREAD

**Issue by asluk** [PixarAnimationStudios/USD#66]
This is because wrapTestPyAnnotatedBoolResult is enabled in the Python bindings, but not in the build. I have a fix.

```
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang++ -arch x86_64 -dynamiclib -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.12.sdk -L/Users/asluk/Desktop/bin/usd-dev/USD/build/pxr/base/lib/tf/Debug -F/Users/asluk/Desktop/bin/usd-dev/USD/build/pxr/base/lib/tf/Debug -filelist /Users/asluk/Desktop/bin/usd-dev/USD/build/pxr/base/lib/tf/usd.build/Debug/_tf.build/Objects-normal/x86_64/_tf.LinkFileList -install_name @rpath/_tf.so -Xlinker -rpath -Xlinker /Users/asluk/Desktop/bin/usd-dev/USD/build/pxr/base/lib/tf/Debug -Xlinker -rpath -Xlinker /Users/asluk/Desktop/bin/usd-dev/USD/build/pxr/base/lib/arch/Debug -mmacosx-version-min=10.11 -Xlinker -object_path_lto -Xlinker /Users/asluk/Desktop/bin/usd-dev/USD/build/pxr/base/lib/tf/usd.build/Debug/_tf.build/Objects-normal/x86_64/_tf_lto.o -Xlinker -no_deduplicate -dynamiclib -Wl,-headerpad_max_install_names /Users/asluk/Desktop/bin/usd-dev/USD/build/pxr/base/lib/tf/Debug/libtf.dylib /Users/asluk/Desktop/bin/usd-dev/USD/build/pxr/base/lib/arch/Debug/libarch.dylib /usr/lib/libm.dylib /Library/Frameworks/Python.framework/Versions/2.7/lib/libpython2.7.dylib /usr/local/lib/libboost_python-mt.dylib /usr/local/lib/libboost_iostreams-mt.dylib /usr/local/lib/libdouble-conversion.dylib /usr/local/lib/libtbb.dylib /usr/local/lib/libtbbmalloc.dylib -single_module -Xlinker -dependency_info -Xlinker /Users/asluk/Desktop/bin/usd-dev/USD/build/pxr/base/lib/tf/usd.build/Debug/_tf.build/Objects-normal/x86_64/_tf_dependency_info.dat -o /Users/asluk/Desktop/bin/usd-dev/USD/build/pxr/base/lib/tf/Debug/_tf.so
```

Undefined symbols for architecture x86_64:
  "wrapTf_TestPyAnnotatedBoolResult()", referenced from:
      WrapModule() in module.o
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)

**Comment by jtran56** [comment]
Filed as internal issue #138122.

**Comment by sunyab** [comment]
Commit has been merged into release v0.7.2, closing.
